styleguide/pane.scss

```diff
@@ -217,6 +229,18 @@ $easeOutExpo: cubic-bezier(.190, 1.000, .220, 1.000);
   }
 }
 
+// special color for warnings
+.kiln-toolbar-pane .publish-warning .label {
+  color: $orange;
```

Comment:
Same comment as above for `$orange`. Feel like we're in super-flux-mode right now so maybe something to keep in back-of-mind.